src/components/PlayerInfo.vue

```diff
@@ -1,7 +1,14 @@
 <template>
   <div :class="['player-info', player.faction]" v-if="player && player.faction" :style="`background-color: ${factionColor}`">
     <div class="text">
-      <b>{{name}}</b> - <span v-b-tooltip.hoover.html="tooltip" >{{faction}}</span> - {{data.victoryPoints}}vp <span v-if="passed">(passed)</span><br/>
+      <b>{{name}}</b> - <span v-b-modal="faction" >{{faction}}</span> - {{data.victoryPoints}}vp <span v-if="passed">(passed)</span>
```

Comment:
Maybe add a `class='faction-name'` to the span with the faction name in it.

Then in the css (at the bottom of the file), add:

```
.faction-name {
  cursor: pointer
}
```

That way, when the mouse goes over the faction name, the pointer changes, indicating that it can be clicked.

Actually, there should also be `role="button"` added for mobile!